orcid-web/src/main/resources/freemarker/includes/ng2_templates/peer-review-ng2-template.ftl

```diff
@@ -63,18 +63,88 @@
                             </ul>                                   
                         </div>
                         <!--End sort menu-->
+
+                        
+                        <ul class="workspace-bar-menu" *ngIf="noLinkFlag">
+                        <!--Add works-->    
+                            <li class="hidden-xs">
+                                <div class="menu-container" id="add-peer-review-container">
+                                    <ul class="toggle-menu">
+                                        <li [ngClass]="{'green-bg' : peerReviewImportWizard == true}"> 
+                                            <span class="glyphicon glyphicon-plus"></span>
+                                            <@orcid.msg 'groups.common.add_peer_review'/>
+                                            <ul class="menu-options works">
+                                            <!--Search & link-->
+                                                <li>
+                                                    <a class="action-option manage-button" (click)="showPeerReviewImportWizard()">
+                                                        <span class="glyphicon glyphicon-cloud-upload"></span>
+                                                        <@orcid.msg 'manual_orcid_record_contents.search_link'/>
+                                                    </a>
+                                                </li>
+                                            </ul>
+                                        </li>
+                                    </ul>
+                                </div>  
+                            </li>  
+                        </ul>    
+                            
                         </#escape>                                
                     </div>
                 </div>
-            </div>                        
+            </div>    
+            <!-- Peer Review Import Wizard -->
+            <div *ngIf="peerReviewImportWizard && workspaceSrvc.displayPeerReview" class="work-import-wizard">
+                <div class="ie7fix-inner">
+                    <div class="row"> 
+                        <div class="col-md-12 col-sm-12 col-xs-12">
+                            <h1 class="lightbox-title wizard-header"><@orcid.msg 'workspace.link_peer_reviews'/></h1>
+                            <span (click)="showPeerReviewImportWizard()" class="close-wizard"><@orcid.msg 'workspace.LinkResearchActivities.hide_link_peer_review'/></span>
+                        </div>
+                    </div>
+                    <div class="row">
+                        <div class="col-md-12 col-sm-12 col-xs-12">
+                            <p class="wizard-content">
+                                <@orcid.msg 'workspace.LinkResearchActivities.description'/>
```

Comment:
@leomendoza123 Please replace workspace.LinkResearchActivities.description text with: 
ORCID works with our member organizations to make it easy to connect your ORCID iD and link to information in their records. Choose one of the link wizards to get started, or contact organizations you perform reviews for to help add reviews to your ORCID record. [More information about peer reviews](https://support.orcid.org/hc/articles/360006971333)